pkg/snet/network.go

```diff
@@ -252,6 +281,22 @@ func (n *Network) IsNetworkReady(netType string) bool {
 	return ok
 }
 
+// WaitTillNetworkReady waits till any one of the network is ready.
+func (n *Network) WaitTillNetworkReady(log *logging.Logger) {
+	n.netsMu.Lock()
+	nets := n.nets
+	n.netsMu.Unlock()
+	if len(nets) == 0 {
+		print(len(nets))
+		time.Sleep(200 * time.Millisecond)
+		n.WaitTillNetworkReady(log)
+	} else {
+		for net := range nets {
+			log.Infof("Transport %v initialized", net)
```

Comment:
This will allow dmsg to allow snet to get initialized. Do we consider dmsg to be sufficient in terms of running a visor @jdknives ?

Reply:
@i-hate-nicknames Either the code here changed or I am missing some context because I cant relate your comment to the code. But I consider dmsg running to be sufficient to let the visor run and start all other modules.